apps/aeutils/src/aeu_requests.erl

```diff
@@ -85,7 +87,7 @@ send_block(Peer, Block) ->
 -spec send_tx(aec_peers:peer(), aec_tx:signed_tx()) -> response(ok).
 send_tx(Peer, SignedTx) ->
     Uri = aec_peers:uri(Peer),
-    TxSerialized = aec_tx_sign:serialize_to_binary(SignedTx),
+    TxSerialized = base64:encode(aec_tx_sign:serialize_to_binary(SignedTx)),
```

Comment:
similar here re base64. Should we introduce e.g. aec_tx_sign:serialize_for_network and make it transparent for Send-Receive handlers?